dbt_subprojects/dex/models/trades/fantom/platforms/yoshiexchange_fantom_base_trades.sql

```diff
@@ -0,0 +1,21 @@
+{{
+    config(
+        schema = 'yoshiexchange_fantom',
+        alias = 'base_trades',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{{
+    uniswap_compatible_v2_trades(
+        blockchain = 'fantom',
+        project = 'yoshiexchange',
+        version = '1',
```

Comment:
Do you think we can use the uniswap fork version `2` here?

Reply:
i think it's fine for a project fork to use their own version. i'm going to approve and merge this